app/models/form/lettings/questions/stock_owner.rb

```diff
@@ -16,20 +16,38 @@ def answer_options(log = nil, user = nil)
     return answer_opts unless log
 
     if log.owning_organisation_id.present?
-      answer_opts = answer_opts.merge({ log.owning_organisation.id => log.owning_organisation.name })
+      answer_opts[log.owning_organisation.id] = log.owning_organisation.name
     end
 
+    recently_absorbed_organisations = user.organisation.absorbed_organisations.merged_during_open_collection_period
     if !user.support? && user.organisation.holds_own_stock?
-      answer_opts[user.organisation.id] = "#{user.organisation.name} (Your organisation)"
+      answer_opts[user.organisation.id] = if recently_absorbed_organisations.exists?
+                                            "#{user.organisation.name} (Your organisation, active as of #{user.organisation.created_at.to_fs(:govuk_date)})"
```

Comment:
This looks like it'll be a bit weird if the absorbing organisation already existed with users before the merge - both because they'd see this and because the date will be further in the past than merged-in users would expect (although presumably that's something we could adjust when we do work around not relying on the created_at date?).

Reply:
Updated this now!